features/config_root.feature

```diff
@@ -0,0 +1,25 @@
+@rspec @disable-bundler
+Feature:
+
+  The root of the project can be customized. A coverage result is considered if
+  an only if it falls inside the root of the project.
+
+  Background:
+    Given I'm working on the project "monorepo"
+    And a file named ".simplecov" with:
+      """
+      SimpleCov.start do
+        root __dir__
+      end
+      """
+
+  Scenario:
+    When I open the coverage report generated with `bin/rspec_binstub_that_chdirs extra/spec/extra_spec.rb`
```

Comment:
I think you might be missing a bundle install step here

Reply:
Yep, didn't notice locally probably because I already had everything installed. Will update.

Reply:
We're using Given I install dependencies elsewhere

Reply:
Still petitioning for a chat tool (2nd time in this PR I wrote my comment without seeing your response first)

![image](https://user-images.githubusercontent.com/606517/85221457-2f767900-b3b4-11ea-899f-627b29c6e20d.png)

Reply:
haha, we can setup a chat tool, sure!

Reply:
No problem about mixed replies though, it happens.

Reply:
That did the trick! :tada: